pkgs/tools/graphics/map-machine/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, python3, pkgs }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "map-machine";
+  version = "unstable-2022-10-26";
+
+  src = fetchFromGitHub {
+    owner = "enzet";
+    repo = pname;
+    rev = "dcf3ffa00902d7812dd581c959e97d8d33f3a78d";
+    sha512 = "sha512-VtdsI6ATIWoksLzUQGEVlcIjQ9ADrBQwfmUPPHI/DHRcC8yKKiiiKuSxF5mOphKIyWVCL065iMdnzRCccI8QLg==";
+  };
+
+  doCheck = false; # Checks require moire, which is not available
+
+  propagatedBuildInputs = with python3.pkgs; [ cairosvg colour numpy pillow pycairo pyyaml setuptools shapely svgwrite urllib3 portolan pytest ];
```

Comment:
propagating pytest is not allowed. Can you please split checkInputs out?

Reply:
Would pushing it to `buildInputs` work? Putting it in `checkInputs` makes the build process fail